section1/reflection.md

```diff
@@ -2,18 +2,45 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+I feel good about asking questions when I need to.  I am the type of learner that really likes to try and figure things out on my own before asking for help, but that being said I can recognize the "unproductive struggle" and I am not afraid to reach out for help if I've spent too long searching for an answer.  I think I land right about in the middle of too soon and too long, with maybe a slight lean towards too long.  So I will try and work on speaking up more if I have trouble and limit my searching to 20-30 minutes.
```

Comment:
That's awesome - it seems like that self-monitoring piece is a strength of yours🙌